react-native-libraries.json

```diff
@@ -9069,5 +9069,16 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/macvish/react-native-basic-carousel",
+    "npmPkg": "@mactony1/react-native-basic-carousel",
```

Comment:
```suggestion
```

Isn't the package name just `react-native-basic-carousel` - https://www.npmjs.com/package/react-native-basic-carousel? If so this field can be skipped.